testing/metaopt/test_random.py

```diff
@@ -0,0 +1,45 @@
+"""
+Tests for Metaopt
+=============================================
+
+..
+    Copyright 2019, Neuraxio Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+"""
+import numpy as np
+from neuraxle.metaopt.random import WalkForwardTimeSeriesCrossValidation
+
+
+def test_walkforward_crossvalidation_split():
+    # Arrange
+    # set random seed
+    np.random.seed(10)
+    data_inputs = np.random.randint(0, high=1, size=(7, 20, 2)).astype(np.float)
```

Comment:
I would extract and name the numbers `7, 20, 2` as `batch_size, time_steps, n_features`.

Reply:
Perfect, will do it.